scripts/filename.js

```diff
@@ -0,0 +1,17 @@
+const fs = require('fs');
+
+function fileName(name, checkAvail = true) {
+    if (name !== undefined) {
```

Comment:
do you use tabstop=4?

Reply:
Yes, I do. Is this a tabstop=2 project? :-)

Reply:
Do your prefer 4 we can make it a mixed project with different tabstop all over the place ;)